planning_dimension : "2D" # available dimension (2D,3D)
planner_name : "CForest" # available planners (CForest, RRTstar)
solving_time : 5.0 # available computation time to attempt to solve the query

#Configuration Space (C-Space) boundaries
planning_bounds_x : [-15, 15]
planning_bounds_y : [-15, 15]
planning_bounds_z : [0, 15]

planning_depth_limit : 1

#Collision Mask
collision_mask_x : 0.5
collision_mask_y : 0.5
collision_mask_z_2D : 0.2
collision_mask_z_3D : 0.5
